docker/base/docker-entrypoint.sh

```diff
@@ -43,19 +43,33 @@ else # nuxeoctl command found
 
   # Handle nuxeo.conf
   if [[ ! -f $NUXEO_HOME/configured && ! -f $NUXEO_CONF ]]; then
-    echo "ENTRYPOINT: Configuration file not found: $NUXEO_CONF"
-    echo "Move $NUXEO_HOME/bin/nuxeo.conf to $NUXEO_CONF"
+    echo "ENTRYPOINT: Server not yet configured and configuration file not provided: $NUXEO_CONF"
```

Comment:
This message is not clear is there a problem or is it a normal case, if it is normal maybe:
Initializing configuration without $NUXEO_CONF